CHANGELOG.rst

```diff
@@ -348,6 +348,7 @@ Minor Changes
 - s3_bucket - Tests for compatability with older versions of the AWS SDKs have been removed (https://github.com/ansible-collections/amazon.aws/pull/442).
 - s3_bucket - add new option ``object_ownership`` to configure object ownership (https://github.com/ansible-collections/amazon.aws/pull/311)
 - s3_bucket - updated to use HeadBucket instead of ListBucket when testing for bucket existence (https://github.com/ansible-collections/amazon.aws/pull/357).
+- s3_bucket - updated module to enable S3 Bucket Keys for SSE-KMS (https://github.com/ansible-collections/amazon.aws/pull/882).
```

Comment:
This file is autogenerated. There's no need to modify this one.

Reply:
I missed that.  @chirag1603 It should be a **fragment** file in changelog/fragments